tests/bsim/bluetooth/host/privacy/legacy/src/tester.c

```diff
@@ -45,8 +47,18 @@ static void cb_expect_rpa(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 
 	/* Only save the address + time if this is the first scan */
 	if (bt_addr_le_eq(&test_data.old_addr, BT_ADDR_LE_ANY)) {
+		int64_t scanner_startup_delay = now - test_data.adv_start_time;
+
+		/* The SDC Controller will miss the first few advertisements due
+		 * to scanner warm-up time. Using the first advertisement report
+		 * as the time reference when the first RPA was generated will
+		 * not be correct. The actual time when the first RPA was
+		 * generated is recorded in `test_data.adv_start_time`.
+		 */
```

Comment:
I am OK with the change, but not sure if we should mentioned specific 3rd party controllers in the Zephyr BSIM tests.

If we keep the reference to the SDC, then I suggest that we spell it out as `Nordic SoftDevice Controller` as people outside of Nordic may be unaware of the `SDC` abbreviation.